ML-Transformers/BackgroundEnchantments/main.ts

```diff
@@ -13,7 +13,7 @@ import { setVonageMetadata } from "@vonage/media-processor";
 import packageInfo from './package.json';
 
 const MEDIA_ASSETS_URI: string =
-  "https://vonage-background-enchantments-sample.s3.amazonaws.com/";
+  "media/";
```

Comment:
We were loading the assets used for video and image background from s3 too so changed that to local media/ route